SETUP.md

```diff
@@ -19,26 +19,46 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 1. `gem install bundler`
 1. `rbenv rehash`
 1. `cd code-dot-org`
-1. `bundle install` (Problems with rmagick? See [tips](#tips) below.) (OS X: when running `bundle install`, you may need to also run `xcode-select --install`. See [stackoverflow](http://stackoverflow.com/a/39730475/3991031))
+1. `bundle install` (Problems running this step? See [tips](#bundle-install-tips) below.) 
 1. `rake install:hooks`
+    <details>
+        <summary>If issues running rake commands</summary>
```

Comment:
This might be outside the scope of your change. It would be nice to standardize these "summary" items. Example:
`<summary>Troubleshoot: 'rake aborted!...'</summary>`

Reply:
Updated!